wal/src/rocks_impl/manager.rs

```diff
@@ -270,40 +270,114 @@ impl RocksImpl {
                 delete_lock: Mutex::new(()),
             };
 
-            let location = WalLocation {
-                versioned_region_id,
-                table_id,
-            };
-            table_units.insert(location, Arc::new(table_unit));
+            table_units.insert(table_id, Arc::new(table_unit));
         }
 
         Ok(())
     }
 
     fn find_table_seqs_from_table_log(
+        &self,
+        table_max_seqs: &mut HashMap<TableId, SequenceNumber>,
+    ) -> Result<()> {
+        self.find_table_seqs_from_table_log_by_region_id(table_max_seqs)
```

Comment:
How about avoiding defining the method `find_table_seqs_from_table_log_by_region_id` because it looks redundant.